riff-raff/app/deployment/Deployments.scala

```diff
@@ -143,6 +125,22 @@ class Deployments(deploymentEngine: DeploymentEngine,
 
   lazy val enableQueueingSwitch = new DefaultSwitch("enable-deploy-queuing", "Enable riff-raff to queue deploys.  Turning this off will prevent anyone queueing a new deploy, although running deploys will continue.", true)
```

Comment:
Based on [this comment](https://github.com/guardian/riff-raff/pull/663/files#r769648481) I think this switch can be removed now?